scheduler/strategy/binpacking.go

```diff
@@ -54,5 +55,16 @@ func (p *BinPackingPlacementStrategy) PlaceContainer(config *dockerclient.Contai
 	// sort by highest weight
 	sort.Sort(sort.Reverse(weightedNodes))
 
-	return weightedNodes[0].Node, nil
+	topWeight := weightedNodes[0].Weight
+	for _, node := range weightedNodes {
```

Comment:
And actually to simplify further how about:

```go
        topNode := weightedNodes[0]
        for _, node := range weightedNodes; node.Weight == topNode.Weight {
                if len(node.Node.Containers()) > len(topNode.Node.Containers()) {
                        topNode = node
                 } 
        }
```

So you don't need to keep another array and sort again